xapian-applications/omega/handler_libetonyek.cc

```diff
@@ -0,0 +1,233 @@
+/** @file handler_libetonyek.cc
+ * @brief Extract text and metadata from Apple documents using libtonyek.
+ */
+/* Copyright (C) 2019 Bruno Baruffaldi
+ *
+ * This program is free software; you can redistribute it and/or
+ * modify it under the terms of the GNU General Public License as
+ * published by the Free Software Foundation; either version 2 of the
+ * License, or (at your option) any later version.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License
+ * along with this program; if not, write to the Free Software
+ * Foundation, Inc., 51 Franklin St, Fifth Floor, Boston, MA  02110-1301
+ * USA
+ */
+#include <config.h>
+#include "handler.h"
+#include "stringutils.h"
+
+#include <memory>
+
+#include <librevenge-generators/librevenge-generators.h>
+#include <librevenge-stream/librevenge-stream.h>
+#include <libetonyek/libetonyek.h>
+
+#define PARSE_FIELD(START, END, FIELD, OUT) \
+   parse_metadata_field((START), (END), (FIELD), (CONST_STRLEN(FIELD)), (OUT))
+
+using std::shared_ptr;
+using libetonyek::EtonyekDocument;
+using namespace librevenge;
+using namespace std;
```

Comment:
Isn't `using std::shared_ptr;` redundant given we then do `using namespace std;`?

Reply:
Also, we don't actually seem to use `shared_ptr` anywhere anyway.

Reply:
Sorry, I am going to remove that.